packit/pkgtool.py

```diff
@@ -39,13 +39,13 @@ def __repr__(self):
             f"tool='{self.tool}')"
         )
 
-    def new_sources(self, sources: Optional[Path] = None, fail: bool = True):
+    def new_sources(self, sources: Optional[Iterable[Path]] = [], fail: bool = True):
```

Comment:
* We need to be careful with using `[]` as a default -- the object is instantiated when parsing and shared between all the calls.
* Not sure what's more readable, but we can use `*args` syntax instead:
```python
pkg_tool_.new_sources(*archives)


def new_sources(self, *sources: Path, fail: bool = True):
    if not sources:
        return
    sources_str = [str(s) for s in source]
```

Reply:
solved. I chose not to use the `*args` syntax.